### ISSUE TITLE
 Collapsible Headings conflicts with Code prettify

### ISSUE THREAD

**Issue by lll9p** [ipython-contrib/jupyter_contrib_nbextensions#767]
When I clicked the code formatting button, the errors below comes out.

```
TypeError: cell is null
http://localhost:6666/nbextensions/collapsible_headings/main.js?v=20161018093041
Line 80
```

firefox49.0.1 
jupyter-contrib-nbextensions (0.2.1)
notebook (4.2.3)

**Comment by jcb91** [comment]
this results from code_prettify attempting to select an undefined cell, as a result of the commented out line at [src/jupyter_contrib_nbextensions/nbextensions/code_prettify/code_prettify.js#L127](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/blob/ca8801be9e66b154e37b9f19652a5a5f0d5a2492/src/jupyter_contrib_nbextensions/nbextensions/code_prettify/code_prettify.js#L127)

**Comment by jfbercher** [comment]
Indeed, there is another error in `code_prettify`; sorry for that. 
There was some code remaining from a previous version where the index of the selected cell was passed to the `yapf_format` function. But actually the index is not defined anywhere anymore. That this actually works most of the time may be because since the index is undefined, then the  select(index) does nothing and [line 130](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/blob/ca8801be9e66b154e37b9f19652a5a5f0d5a2492/src/jupyter_contrib_nbextensions/nbextensions/code_prettify/code_prettify.js#L130) selects the correct cell. 
I have just removed the reference to the index and corrected the code accordingly. 
Tested on my configuration. Hope it will fix this.

**Comment by jcb91** [comment]
thanks @jfbercher, I think your diagnosis seems correct, and usually it works because selecting an undefined cell isn't a problem. As a result, I think it makes sense to fix this on the `collapsible_headings` side as well, as @lll9p has done in #768

**Comment by jcb91** [comment]
should now be fixed by #768, #770
